### PR TITLE
grpc-service: Add_split_and_scatter_region

### PR DESCRIPTION
**Pull request by hzh0425**
### What problem does this PR solve?

Close pingcap/tidb#29034

### What is changed and how it works?

Add a method splitAndScatterRegions in grpc-service

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
 Let pd handle split and scatter regions directly
```

### CODE REVIEW

**Comment by hzh0425** [comment]
@Yisaer @disksing  plz approve again, thanks!!

**Review by Yisaer — APPROVED** [review]

**Comment by Yisaer** [comment]
plz fix the release message and keep `None`

File: client/client.go [hunk 1]
```diff
@@ -1695,6 +1697,32 @@ func (c *client) scatterRegionsWithOptions(ctx context.Context, regionsID []uint
 	return resp, nil
 }
 
+func (c *client) SplitAndScatterRegions(ctx context.Context, splitKeys [][]byte, opts ...RegionsOption) (*pdpb.SplitAndScatterRegionsResponse, error) {
```
Comment (JmPotato) [inline client/client.go:1700]: A comment is needed.

**Comment by rleungx** [comment]
Please add some tests to cover these codes.

**Comment by JmPotato** [comment]
@hzh0425 Hi, could you please help finish this PR?

**Comment by hzh0425** [comment]
> @hzh0425 Hi, could you please help finish this PR?

Ok, I'll finish it recently

**Comment by hzh0425** [comment]
@JmPotato Sorry, I've delayed this pr...
I have a question. Where should I put the test? I found Splitregion and scatterregion in the grpc_service also have no related tests

**Review by disksing — APPROVED** [review]

**Comment by JmPotato** [comment]
> @JmPotato Sorry, I've delayed this pr... I have a question. Where should I put the test? I found Splitregion and scatterregion in the grpc_service also have no related tests

Maybe you could put some tests in the https://github.com/tikv/pd/blob/master/tests/client/client_test.go, it is used to test the PD client which is a proper way to test the new gRPC methods.

**Pull request closed without merging**